wwwroot/test/AssImp/README.md

```diff
@@ -0,0 +1,19 @@
+# Assimp test files
```

Comment:
Do we want to include these?

Reply:
Yep, these files are used in the test file https://github.com/TerriaJS/terriajs/blob/146aaa4d1db399d8accb3bb0e69d479b8d2e410e/test/Models/Catalog/CatalogItems/AssImpCatalogItemSpec.ts